.soliumrc.json

```diff
@@ -6,7 +6,6 @@
     "no-empty-blocks": "off",
     "indentation": ["error", 2],
     "max-len": ["error", 79],
-    "arg-overflow": ["warning", 3],
```

Comment:
This is one of the default rules, so we need to explicitly disable it:

`"arg-overflow": "off"`

Reply:
Maybe we could add some sample code to this repo, showcasing the style (and checking that our ruleset works with no warnings).